configs/common/packages.yaml

```diff
@@ -62,16 +62,17 @@
       version: ['8.4.2']
       variants: ~xerces ~pnetcdf snapshot=none ~shared +external-parallelio
     fckit:
-      version: ['0.10.1']
+      version: ['0.11.0']
       variants: +eckit
     fftw:
       version: ['3.3.10']
     fiat:
-      version: ['1.1.0']
+      version: ['1.2.0']
     fms:
-      version: ['2023.01']
-      # when switching to 2023.02, need to add "+use_fmsio" to variants
-      variants: precision=32,64 +quad_precision +gfs_phys +openmp +pic constants=GFS build_type=Release
+      #version: ['2023.01']
+      #variants: precision=32,64 +quad_precision +gfs_phys +openmp +pic constants=GFS build_type=Release
+      version: ['2023.02']
+      variants: precision=32,64 +quad_precision +gfs_phys +openmp +pic constants=GFS build_type=Release +use_fmsio
```

Comment:
jun's comment is addressed via the `use_fmsio` variant here, correct? (i.e.: https://github.com/ufs-community/ufs-weather-model/issues/1874#issuecomment-1683824237).

Reply:
I think Jun is hoping to not having to use the deprecated fms_io component, but as she mentioned there is work that needs to be done. JEDI also needs fms_io. By enabling it (for now), we can simplify and speed up the transition to that version and abandon the old release-jcsda release.